run_workflow.py

```diff
@@ -331,6 +331,9 @@ def start_workflow(link_id, site_id, APP):
                     if 'state' in step:
                         state = step['state']
 
+                    if 'flagged' in step and not step['flagged']:
```

Comment:
What is this doing exactly?

Reply:
Hey @smarquard, that is correct, it is like a feature flag. Just a flag to help us decide whether to run a workflow step or not to.